.github/dependabot.yml

```diff
@@ -0,0 +1,39 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    # Look for a `requirements` in the `root` directory
+    directory: "/"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
+    # Labels on pull requests for version updates only
+    labels: ["maintenance"]
+    pull-request-branch-name:
+      # Separate sections of the branch name with a hyphen
+      # for example, `dependabot-npm_and_yarn-next_js-acorn-6.4.1`
+      separator: "-"
+    # Allow up to 5 open pull requests for pip dependencies
+    open-pull-requests-limit: 5
+    reviewers:
+      - "hugovk"
```

Comment:
Please remove me here and below. I can merge if I see them, but they're not urgent and I get enough pings already :)

Reply:
sure and remove just here or from all (pip + gh)?

Reply:
Both please.

Reply:
done